src/qt/walletcontroller.cpp

```diff
@@ -206,7 +207,7 @@ void CreateWalletActivity::askPassphrase()
 
 void CreateWalletActivity::createWallet()
 {
-    showProgressDialog(tr("Creating Wallet <b>%1</b>...").arg(m_create_wallet_dialog->walletName().toHtmlEscaped()));
+    showProgressDialog(tr("Creating Wallet"), tr("<b>%1</b>").arg(m_create_wallet_dialog->walletName().toHtmlEscaped()));
```

Comment:
`tr("<b>%1</b>")`

https://github.com/bitcoin/bitcoin/pull/17978#discussion_r369719589 by @MarcoFalke:
> html code should not be translated

Reply:
A make clean fixed the original UI issue on my end.

I will remove the bold brackets from the commit.

Before the next push

I am going to take another look at the progress bar issue for Mac.
I suspect the progress bar isn’t inheriting the size hinting and/or the vertical/horizontal expand flag by default - as it is on Linux.

Reply:
Should really be translating the Wallet Name either right?

Reply:
> I will remove the bold brackets from the commit.

Just append these at before and after the translated string